tests/jobs.py

```diff
@@ -19,12 +19,14 @@
 
```

Comment:
Technically, no, because these functions are called in run(), which is called in a bunch of other places, and as you can see from my 18 commits, I broke a lot of tests initially. But i was probably gonna add some tests for the run_local (the pre_run if you will), since it returns True or False